cmd/commands/service.go

```diff
@@ -324,19 +325,21 @@ The curl command is printed so it can be copied and extended.
 
 Additional curl arguments and flags may be specified after a double dash (--).`,
 		Example: `  riff service invoke square --namespace joseph-ns
-  riff service invoke square -- --include`,
+  riff service invoke square /foo -- --data 42`,
 		Args: UpToDashDash(ArgValidationConjunction(
-			cobra.ExactArgs(serviceInvokeNumberOfArgs),
+			cobra.MinimumNArgs(serviceInvokeMaxNumberOfArgs - 1),
+			cobra.MaximumNArgs(serviceInvokeMaxNumberOfArgs),
 			AtPosition(serviceInvokeServiceNameIndex, ValidName()),
 		)),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			serviceInvokeOptions.Name = args[serviceInvokeServiceNameIndex]
+			path := args[serviceInvokeServicePathIndex]
```

Comment:
This doesn't support the case where the arg is not set, does it?

Reply:
I'll check that with the test I should have added 😓